openssl.sh

```diff
@@ -25,18 +38,23 @@ rsync -av --delete --exclude="**/.git" $SOURCEDIR/ .
          no-ecdsa                                  \
```

Comment:
It seems like one of the three lines is responsible for the failing build of grpc in the o2 builds. If I remove these lines grpc builds.